lib/ohai/plugins/aix/os.rb

```diff
@@ -25,6 +25,6 @@
 
   collect_data(:aix) do
     os collect_os
-    os_version kernel[:version]
+    os_version shell_out("oslevel -s").stdout.split($/)[0]
```

Comment:
It's to hedge against whitespace appearing after the command output. Sometimes IBM utilities are not clean on this front.